src/components/MapPoiTooltip/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC } from 'react'
+import { Popup } from 'react-map-gl'
+
+export interface MapPoiTooltipType {
+  title?: string
+  category?: string
+  info?: string
+  coordinates: {
+    latitude: number
+    longitude: number
+  }
+}
+
+export const MapPoiTooltip: FC<MapPoiTooltipType> = ({
+  title,
+  category,
+  info,
+  coordinates,
+}) => {
+  return (
+    <Popup
+      latitude={coordinates.latitude}
+      longitude={coordinates.longitude}
+      closeButton={false}
+      className="w-64 rounded-2xl"
```

Comment:
Feel free to ignore, but I'm not convinced about the aesthetic of the hugely rounded corner. It is not really matching the rest of the UI. Also, I think increasing the tooltip's padding should make its content breathe a little better.

Reply:
Good points. I adjusted that.